src/compiler/transformers/utilities.ts

```diff
@@ -269,8 +269,7 @@ namespace ts {
      * any such locations
      */
     export function isSimpleInlineableExpression(expression: Expression) {
-        return !isIdentifier(expression) && isSimpleCopiableExpression(expression) ||
-            isWellKnownSymbolSyntactically(expression);
+        return !isIdentifier(expression) && isSimpleCopiableExpression(expression);

```

Comment:
This change is pretty much single-handedly responsible for the .js emit changes in this PR. `Symbol.iterator` and it's ilk aren't "simple copyable expressions" since, technically, accessing it could produce side effects (like any other property access). Now... at some point, we added `Identifier`s to `isSimpleCopiableExpression`'s domain (which isn't technically safe, since the identifier may get reassigned between the copied references), so maybe property accesses which only consist of `Identifier`s wouldn't be terribly unreasonable, either. 🤷

Reply:
@rbuckton is probably the best judge of that.